app/models/concerns/product/structured_data.rb

```diff
@@ -39,4 +39,11 @@ def build_book_work_examples
         work_example
       end
     end
+
+    def product_description
+      (custom_summary.presence || strip_tags(html_safe_description).presence)
+        .to_s
+        .truncate(160)
+        .presence
+    end
```

Comment:
Thanks for catching this and putting up a fix! I hadn’t considered that edge case. Let me know if I can help test or refine anything.